sdk/core/azure_core_amqp/src/connection.rs

```diff
@@ -81,45 +124,54 @@ pub mod builders {
                 options: Default::default(),
             }
         }
-        pub fn build(self) -> AmqpConnectionOptions {
-            self.options
+        pub fn build(&mut self) -> AmqpConnectionOptions {
+            self.options.clone()
         }
         #[allow(dead_code)]
```

Comment:
Are all these annotations needed now? I'm guessing they are left-overs.

Reply:
Removed. When I initially built this, Rust was upset with them, but it doesn't seem to be as annoyed any more.